packages/server/src/server.ts

```diff
@@ -47,14 +49,18 @@ function verifyToken(request: Request, response: Response, next: NextFunction) {
 
 app.use(verifyToken)
 
+const dbConnection = DbConnection.getInstance()
```

Comment:
having both `dbConnection` and `databaseConnection` is a bit confusing. What is the difference among the two?

Reply:
Renamed them all and added comments to explain them.